docker/workspace-build/Dockerfile

```diff
@@ -23,6 +23,10 @@ RUN apt-get -y update \
         python3-venv \
         ros-${ROS_DISTRO}-image-transport-plugins \
         ros-${ROS_DISTRO}-rmw-cyclonedds-cpp \
+        ros-${ROS_DISTRO}-ros2bag \
+        ros-${ROS_DISTRO}-rosbag2-transport \
```

Comment:
If we are building these packages as part of the rosbag2 suite we've pulling in via the git repo, do we still need these installs here?